### ISSUE TITLE
CMSP line 1 doesn't display on LCD but line 2 and text will display properly.

### ISSUE THREAD

**Issue by Mattack-1412** [dcs-bios/dcs-bios#87]
I'm having a weird problem my LCD will display everything except CMSP line 1 UNLESS I take out the other code that I have. I have an Arduino Mega 2560 R3 by OSEPP. Here is my code.

```c++
/* use '#define DCSBIOS_DEFAULT_SERIAL' instead if your Arduino board
 *  does not feature an ATMega328 or ATMega2650 controller.
 */
#define DCSBIOS_IRQ_SERIAL
#include "DcsBios.h"
#include <LiquidCrystal.h>
LiquidCrystal lcd(8, 9, 4, 5, 6, 7);

// These lines were commented out to make it show the first line
/* Declare a Master Caution Reset button on pin 10 */
//DcsBios::Switch2Pos masterCautionBtn("UFC_MASTER_CAUTION", 22);
/* Make the LED connected to pin 13 into a Master Caution Light */
//DcsBios::LED mcLed(0x1012, 0x0800, 23);


//CMSP
void onCmsp1Change(char* newValue) {
    /* your code here */
    lcd.setCursor(0, 0);
    lcd.println(newValue);
}
DcsBios::StringBuffer<19> cmsp1Buffer(0x1000, onCmsp1Change);

void onCmsp2Change(char* newValue) {
    /* your code here */
    lcd.setCursor(0,1);
    lcd.print(newValue);
}
DcsBios::StringBuffer<19> cmsp2Buffer(0x1014, onCmsp2Change);
void setup() {
  DcsBios::setup();
  lcd.begin(16,2);
  lcd.clear();
  lcd.setCursor(0,0);
  lcd.print("Running");
  lcd.setCursor(0,1);
  lcd.print("Program");
}
void loop() {
  DcsBios::loop();
}

```

**Comment by Mattack-1412** [comment]
I also tried putting the LED and Button code below the LCD but it did not fix it.

**Comment by Mattack-1412** [comment]
It is actually just the LED code

**Comment by Mattack-1412** [comment]
Tried moving the pin as well

**Comment by jboecker** [comment]
I have reproduced the issue here and will look into it. Thanks for the report!

By the way, GitHub does not use BBCode, so the `[code]` tag does not work. You can read about how to format code blocks on GitHub here:
https://help.github.com/articles/creating-and-highlighting-code-blocks/
I took the liberty of fixing that in your first post so I could easily copy and paste the sketch.

**Comment by jboecker** [comment]
Fixed in the new v0.2.3 release of the Arduino library.

**Comment by Mattack-1412** [comment]
Thanks
